_search-plugins/knn/knn-index.md

```diff
@@ -60,14 +60,19 @@ SIMD optimization is applicable only if the vector dimension is a multiple of 8.
 ### x64 architecture
 <!-- vale on -->
 
-For the x64 architecture, two different versions of the Faiss library are built and shipped with the artifact:
+For the x64 architecture, three different versions of the Faiss library are built and shipped with the artifact:
```

Comment:
Shall we add a note showing the order of performance avx512>avx2>generic ?

Reply:
I added a note specifying order of performance below the list of libraries (line 69). Let me know if you think it would be better directly after line 63 instead.

Reply:
It needs to be added as a note. @kolchfa-aws can you help to fix it when you are reviewing it later

Reply:
Sure, will do.